### PR TITLE
Changed some descriptions in Polish lang file

### PR DESCRIPTION
**Pull request by Greg-21**
In this commit I changed some descriptions in Polish lang file and included a (small) part of changes made in #75.

### CODE REVIEW

**Comment by Greg-21** [comment]
Additional thing I'd like to discuss here is the inconsistency of the descriptions in `sidebar.loading` and `sidebar.versions` strings. The first one contains "wczytywanie" while the second – "ładowanie".

**Comment by Greg-21** [comment]
If you want to send me a private message once or only a few times, I can publish my 3rd e-mail. Why only a few times? Because I'm using this e-mail account very rare. Its main purpose is to use it on new services, untested by me yet. To avoid unnecessary spam to my 2nd (or 1st) e-mail. The second option is Facebook/Messenger, but my account on it is too personal.

**Comment by Greg-21** [comment]
I just resolved the conflicts. And I'm going to made the requested changes (all in one commit), but still waiting for @Waterrail review since the Polish translation it's mostly his.

**Comment by Greg-21** [comment]
I just resolved the conflicts. And I'm going to made the requested changes (all in one commit), but still waiting for @Waterrail review since the Polish translation is mostly his.

File: assets/launcher/lang/sklmessages_pl_PL.properties [hunk 1]
```diff
@@ -40,7 +40,7 @@ game.output.type.text=tekstowy
 
 game.conflict.title=Ostrzeżenie o zduplikowanej instancji
 game.conflict.header=Posiadasz już uruchomioną instancję Minecrafta.
-game.conflict.text=Jeśli uruchomisz kolejną z tego samego folderu, może dojść do konfliktu i uszkodzenia twoich zapisów gry. \nMoże to stworzyć wiele problemów, w grze jednoosobowej lub w inny sposób. Nie będziemy ponosić odpowiedzialności za cokolwiek, co pójdzie nie tak. \nCzy mimo to chcesz uruchomić kolejną instancję Minecrafta? \nMoże udać ci się zapobiec problemom, jeżeli uruchomisz grę w innym folderze (patrz: przycisk "Edytuj profil").
+game.conflict.text=Jeśli uruchomisz kolejną z tego samego katalogu, może dojść do konfliktu i uszkodzenia Twoich zapisów gry. \nMoże to stworzyć wiele problemów, w grze jednoosobowej lub w inny sposób. Nie będziemy ponosić odpowiedzialności za cokolwiek, co pójdzie nie tak. \nCzy mimo to chcesz uruchomić kolejną instancję Minecrafta? \nMoże uda Ci się zapobiec problemom, jeżeli uruchomisz grę w innym katalogu (patrz: przycisk "Edytuj profil").
```
Comment (Waterrail) [inline assets/launcher/lang/sklmessages_pl_PL.properties]: Ja bym zbytnio nie przesadzał z dużymi literami i grzecznością, to w końcu launcher, a nie list

File: assets/launcher/lang/sklmessages_pl_PL.properties [hunk 2]
```diff
@@ -62,18 +62,18 @@ editor.keep=Pozostaw launcher otwarty
...
 editor.compatibility.alert.title=Tryb zgodności
-editor.compatibility.alert.text=Ta opcja może być wymagana w przypadku niektórych paczek modów, chociaż w większości przypadków nie jest. Włącz ten tryb tylko, jeśli wiesz, co robisz!
-editor.advanced=Zaawansowane ustawienia
-editor.exe=Plik wykonywalny
+editor.compatibility.alert.text=Ta opcja może być wymagana w przypadku niektórych, specyficznych paczek modów, ale w większości przypadków nie jest. Włącz ten tryb tylko wtedy, gdy wiesz, co robisz!
+editor.advanced=Ustawienia zaawansowane
+editor.exe=Plik wykonywalny Javy
 editor.jvm=Argumenty JVM
-editor.alert.title=Potwierdzenie profilu
+editor.alert.title=Potwierdzenie usuwania profilu
```
Comment (Waterrail) [inline assets/launcher/lang/sklmessages_pl_PL.properties]: Może _Potwierdzenie usunięcia profilu_?

File: assets/launcher/lang/sklmessages_pl_PL.properties [hunk 3]
```diff
@@ -98,7 +98,7 @@ crash.modded=Przypuszczamy, że Twoja gra posiadała mody, więc nie możemy zaa
 crash.open=Otwórz raport
 
 crash.alert.title=Ups! Gra się scrashowała
-crash.alert.text=Znaleźliśmy za to możliwe rozwiązanie twojego problemu. Czy chcesz na nie zerknąć?
+crash.alert.text=Znaleźliśmy za to możliwe rozwiązanie Twojego problemu. Czy chcesz je teraz sprawdzić?
```
Comment (Waterrail) [inline assets/launcher/lang/sklmessages_pl_PL.properties]: Tu też duża litera

**Comment by Waterrail** [comment]
@Greg-21 Please review

**Comment by Greg-21** [comment]
OK. I made the suggested changes. Please give me a while to upload them.

**Comment by Greg-21** [comment]
OK. I made the suggested changes. Feel free to suggest more :)

**Comment by Waterrail** [comment]
I think it's done now.

**Comment by Greg-21** [comment]
I'd prefer to firstly rebase and then merge it. But SKmedix often just merges changes.

**Comment by Greg-21** [comment]
I'd prefer to firstly rebase it and then merge it. But SKmedix often just merges changes.

**Pull request merged**